src/components/AddRecipe.js

```diff
@@ -0,0 +1,112 @@
+import React, { useState } from 'react';
+import RecipeService from '../services/RecipeService';
+
+const AddRecipe = () => {
+  const initialRecipeState = {
```

Comment:
Given that this variable never changes, it's better to place it outside of the `AddRecipe` component. That way, we don't initialize it on every render.